xml/ubl21/example/ubl21_invoice.xml

```diff
@@ -1,58 +1,530 @@
-<Invoice xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2"
-         xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2"
-         xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2">
-    <cbc:ID>123</cbc:ID>
+<Invoice
```

Comment:
it already is in `ubl21` folder so we can omit this prefix from file name
and also pls rename folder to `ubl2.1`